index.bs

```diff
@@ -3036,6 +3036,14 @@ optionally evidence of [=user consent=] to a specific transaction.
         The [=client=] and [=authenticator=] make a best-effort to create a credential of the most preferred type possible.
         If none of the listed types can be created, the {{CredentialsContainer/create()}} operation fails.
 
+        [=[RP]=]'s that wish to support a wide range of [=authenticators=] SHOULD define entries for the following discrete list of signature algorithms:
```

Comment:
"SHOULD define entries for" reads a bit awkward to me. Perhaps something like this would read better?

`SHOULD include at least the following algorithm identifiers:` (or even `the following {{COSEAlgorithmIdentifier}} values:`)

With that we could probably remove the trailing "Additional signature algorithms can be included as needed" as well.

Reply:
Great suggestions, thank you.